src/spatial_ref/srs.rs

```diff
@@ -65,19 +65,25 @@ impl PartialEq for SpatialRef {
 }
 
 impl SpatialRef {
-    pub fn new() -> SpatialRef {
+    pub fn new(definition: &str) -> Result<SpatialRef> {
```

Comment:
Is there no use-case for `new()` without parameters? Maybe the wrapped `OSRSetFromUserInput` should be a separate method like `new_from_definition(definition: &str)` or something like this?